### PR TITLE
Remove permissions not used by the operator

### PR DESCRIPTION
**Pull request by david-kow**

### CODE REVIEW

**Comment by thbkrkr** [comment]
Jenkins test this please

**Comment by david-kow** [comment]
run full pr build

**Comment by thbkrkr** [comment]
Jenkins test this please

File: deploy/eck-operator/templates/_helpers.tpl [hunk 1]
```diff
@@ -223,8 +222,6 @@ RBAC permissions
   - elasticsearches
   - elasticsearches/status
   - elasticsearches/finalizers # needed for ownerReferences with blockOwnerDeletion on OCP
-  - enterpriselicenses
-  - enterpriselicenses/status
```
Comment (pebrc) [inline deploy/eck-operator/templates/_helpers.tpl:227]: 😮

**Review by pebrc — COMMENTED** [review]
What about delete permissions on the CRDs? Is there a reason to keep those?

**Comment by david-kow** [comment]
> What about delete permissions on the CRDs? Is there a reason to keep those?

Not sure if I understand, we do watch/get/update them and their statuses. Or you've meant thinning down the verbs?

**Comment by pebrc** [comment]
> > What about delete permissions on the CRDs? Is there a reason to keep those?
> 
> Not sure if I understand, we do watch/get/update them and their statuses. Or you've meant thinning down the verbs?

Yes removing the delete verb was what I was thinking of.

**Comment by david-kow** [comment]
> > > What about delete permissions on the CRDs? Is there a reason to keep those?
> > 
> > 
> > Not sure if I understand, we do watch/get/update them and their statuses. Or you've meant thinning down the verbs?
> 
> Yes removing the delete verb was what I was thinking of.

Oh, of course, I read it as "deleting CRD permissions", my bad.

File: docs/operating-eck/eck-permissions.asciidoc [hunk 2]
```diff
@@ -0,0 +1,150 @@
+:page_id: operator-config
+ifdef::env-github[]
```
Comment (pebrc) [inline docs/operating-eck/eck-permissions.asciidoc:2]: This page seems identical to the existing `operator-config` page. Also it does not seem to be linked anywhere. Is that maybe a file you forgot to delete?

**Review by pebrc — CHANGES_REQUESTED** [review]
I think something is off with the doc page you added. Maybe a Git mishap?

File: docs/operating-eck/eck-permissions.asciidoc (hunk 2 above)
Comment (david-kow) [reply docs/operating-eck/eck-permissions.asciidoc]: Yes, git mishap, thanks for noticing.

**Review by pebrc — APPROVED** [review]

**Pull request merged**